### PR TITLE
JIRAID: Substraction feature is created

### PR DESCRIPTION
**Pull request by ghost**
Substraction feature is added.

Can you please review.

### CODE REVIEW

File: Mayuri.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Different features are created here...
```
Comment (yogskarma) [inline Mayuri.txt:1]: This is file is not required.
Please delete the file.

File: login1.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Public void sub(){
```
Comment (yogskarma) [inline login1.txt:1]: filename and method name does not match.
Please change either of it. Also change branch name accordingly.

File: login1.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+  int a ,b,c;
+	c=a+b;
+
+
+
```
Comment (yogskarma) [inline login1.txt:6]: Please ensure code is well-formatted.

**Review by yogskarma — COMMENTED** [review]
Please do change as per comments

File: Mayuri.txt (hunk 1 above)
Comment (ghost) [reply Mayuri.txt:1]: I will do it

**Pull request closed without merging**